app/components/autocomplete/slash_suggestion/app_command_parser/app_command_parser.ts

```diff
@@ -1538,10 +1532,9 @@ export class AppCommandParser {
     };
 
     public getSubmittableForm = async (location: string, binding: AppBinding): Promise<{form?: AppForm; error?: string} | undefined> => {
-        const manager = IntegrationsManager.getManager(this.serverUrl);
         const rootID = this.rootPostID || '';
         const key = `${this.channelID}-${rootID}-${location}`;
-        const submittableForm = this.rootPostID ? manager.getAppRHSCommandForm(key) : manager.getAppCommandForm(key);
+        const submittableForm = AppsManager.getCommandForm(this.serverUrl, key, Boolean(this.rootPostID));
```

Comment:
Are we able to follow a similar pattern of the webapp to have the dependencies for the command parser be defined in an adjacent file `app_command_parser_dependencies.ts`, in order to keep the contents of `app_command_parser.ts` in sync with the webapp's version? Or has the implementation drifted to the point where this isn't feasible?

Reply:
I have done it "fast", to make sure it is in a working state. Now that it is in a working state, we should be able to take again both web and mobile, and do a new review on how to make them sync-able. I don't think it has drifted too far away, but having different kind of store (redux vs watermelondb + manager), we may have to move more things out of the common part.

I would prefer to leave that outside of the scope of this PR.